RecoTracker/LST/plugins/alpaka/LSTProducer.cc

```diff
@@ -77,14 +81,16 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
       desc.add<edm::InputTag>("pixelSeedInput", edm::InputTag{"lstPixelSeedInputProducer"});
       desc.add<edm::InputTag>("phase2OTHitsInput", edm::InputTag{"lstPhase2OTHitsInputProducer"});
       desc.add<int>("verbose", 0);
+      desc.add<int>("nopLSDupClean", 0);
+      desc.add<int>("tcpLSTriplets", 0);
```

Comment:
why `int` ?

Reply:
Consistency with `verbose_` (which is not well justified). But I can take the opportunity and move everything to `bool` in this PR.

Reply:
verbosity is frequently multi-leveled (from quiet to `very-*-very` verbose). The yes/no options are not.

Reply:
OTOH, `verbose` is a `bool` in the `LST::run`. So, makes sense to keep them all `bool`

Reply:
If we decide to go with a more elaborate verbosity scheme, let's switch back to the appropriate type. For now, I switched everything to `bool`, as proposed.